dcos_installer/backend.py

```diff
@@ -127,8 +126,32 @@ def calculate_cloudformation_s3_url(bootstrap_url, config_id):
     return '{}/config_id/{}'.format(bootstrap_url, config_id)
 
 
+def calculate_aws_template_storage_region_name(
+        aws_template_storage_access_key_id,
+        aws_template_storage_secret_access_key,
+        aws_template_storage_bucket):
+
+    if not aws_template_storage_access_key_id:
```

Comment:
nit: These would be better as single-parameter validation functions (Single parameter ones are guaranteed to always be evaluated before any multi-parameter ones. There are two "phases" more or less)